src/Config/Consumer.php

```diff
@@ -38,9 +38,15 @@ public function __construct(
         string $topic,
         string $consumerGroupId = null,
         int $partition = null,
-        int $offset = null
+        int $offset = null,
+        string $brokerKey = null
     ) {
         parent::__construct($topic);
```

Comment:
You could pass the `$brokerKey` here and then when calling `setBroker` perhaps `$this->setBroker($brokerKey ?: $topicConfig['broker']);`